java/cdk/src/main/java/sleeper/cdk/stack/DynamoDBStateStoreStack.java

```diff
@@ -47,25 +47,25 @@ public DynamoDBStateStoreStack(Construct scope, String id, InstanceProperties in
         RemovalPolicy removalPolicy = removalPolicy(instanceProperties);
 
         // DynamoDB table for active file information
-        Attribute partitionKeyActiveFileInfoTable = Attribute.builder()
+        Attribute partitionKeyActiveFileReferenceTable = Attribute.builder()
                 .name(DynamoDBStateStore.TABLE_ID)
                 .type(AttributeType.STRING)
                 .build();
-        Attribute sortKeyActiveFileInfoTable = Attribute.builder()
+        Attribute sortKeyActiveFileReferenceTable = Attribute.builder()
                 .name(DynamoDBStateStore.PARTITION_ID_AND_FILENAME)
                 .type(AttributeType.STRING)
                 .build();
 
-        activeFileInfoTable = Table.Builder
-                .create(this, "DynamoDBActiveFileInfoTable")
+        activeFileReferenceTable = Table.Builder
+                .create(this, "DynamoDBActiveFileReferenceTable")
```

Comment:
It might be worth making this just DynamoDBActiveFilesTable?

Reply:
Done, thanks